src/modules/updateLocalStorage.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/no-cycle
```

Comment:
- [ ] [OPTIONAL] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: